### DISCUSSION TITLE
Calico BGP peer stuck in Established state.

### DISCUSSION THREAD

**Discussion by alessandro-veras** [projectcalico/calico#8439] [Q&A]
Hi, Calico community!!!!
We have a K8s deployment running Calico and we run BGP from the Node to two leaf switches in the rack. 
When we reboot one of the switches the bgp session never goes down and the node keeps using the unavailable next-hop. 
This is causing downtime as traffic only works again after the switch is up.

Hope someone here can offer some light on what might be happening.

**Comment by caseydavenport** [comment]
When you restart the swtich, are you restarting the entire routing stack as well or just the L2 switch component of the device? I wonder if the latter, that the BGP control plane stays established but the data plane isn't functional during that time. If so, bringing the BGP interface / protocol down administratively on the router before restart might do the trick?
